kilkari/src/main/java/org/motechproject/nms/kilkari/web/MctsBeneficiaryImportController.java

```diff
@@ -63,6 +63,11 @@ public void setCsvAuditService(CsvAuditService csvAuditService) {
     @ResponseStatus(HttpStatus.OK)
     public void importMotherData(@RequestParam MultipartFile csvFile) {
 
+        // Mcts mother import from csv is not supported
+        if(true) {
```

Comment:
Not a fan of this coding style. If you are really trying to remove this, you should do it the right way instead of an always true condition and leaving coding artifacts around. This feels like an easy-lazy solution that is not good design.